### ISSUE TITLE
macOS Build Automation

### ISSUE THREAD

**Issue by rkeene** [nanocurrency/raiblocks#1250]
Verify rpath (collect error from users)
Use SDK for minimum version of macOS we want to support (which is... ?)

**Comment by argakiig** [comment]
the rpath can be fixed by using the opensource qt_everywhere installer to get 5.9.x IIRC

**Comment by rkeene** [comment]
The CMake file tries to set the RPATH but it doesn't seem to work, so need to find out why

**Comment by rkeene** [comment]
Also related, #1185

**Comment by rkeene** [comment]
We're going to minimum version of either 10.9 or 10.11, looking into which our userbase cares about

**Comment by rkeene** [comment]
We're going to support Mac OS X 10.11+ based on the user feedback and considering Apple's support lifecycle.

I looked at the CMake stuff and it only sets the RPATH on the installed files, so it modifies them after installation and I verified it is done correctly.  The incorrect part is that each shared object/dynamic library specifies an absolute path to each dependent object.  The `install_name_tool` can update this, but rather than resorting to that we'll just build the version of Qt5 we intend to support as we do with Boost.
